### ISSUE TITLE
Does not handles strings containing "\u"

### ISSUE THREAD

**Issue by OscarL** [timothycrosley/deprecated.frosted#14]
On Python 2.7, Windows XP.

If any string on the file to be checked by frosted contains "\u" anywhere inside it, frosted fails to analise the file.

Save the following as test.py

```
"""
Example: C:\foobar\unit-tests\test.py
"""
```

Run frosted on it, and you get:

```
test.py:3: (unicode error) 'unicodeescape' codec can't decode bytes in position 19-21: truncated \uXXXX escape
"""
```

Comments are not affected by this, but non docstrings are:

Save the following as test2.py

```
foo = r'.\units'
```

Run frosted on it, and you get:

```
test2.py:1: (unicode error) 'rawunicodeescape' codec can't decode bytes in position 1-2: truncated \uXXXX
foo = r'.\units'
```

**Comment by timothycrosley** [comment]
Thanks for reporting! Will release a new version soon to resolve this.

**Comment by timothycrosley** [comment]
Fixed in release 1.2.0, thanks again for reporting!

**Comment by OscarL** [comment]
Just checked it, now works like a charm. Thanks for your work!
